data/AndroGari.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "Garima Saroj",
+  "tag": "SIH'22 Winner | Ex Jr. ML Engineer @JupiterAILabs | CSE(AI&ML) '24"
```

Comment:
Updating on it because the issue wasn't resolved, but it was marked resolved,

```suggestion
```

Please have a look at the docs for more info